lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -183,7 +183,7 @@ public function trackLastInsertId() : void
      *
      * @return string The last insert ID or '0' in case the last insert ID generated on this connection is unknown.
      *
-     * @throws PDOException
+     * @throws \PDOException
```

Comment:
Exception type changed to base exception here, since the wrapped connection is accessed directly.